portfolio/src/main/java/com/google/sps/Task.java

```diff
@@ -0,0 +1,15 @@
+package com.google.sps;
+
+/** An item on a todo list. */
+public final class Task {
+
+  private final long id;
+  private final String title;
+  private final long timestamp;
+
+  public Task(long id, String title, long timestamp) {
```

Comment:
Does this need to have a field called "comment" since you're setting it in the other file?

Reply:
It doesn't, originally the example uses "title" as the String name it passes but I changed it to comment to reflect what the String actually was. In Task.java I left it as title since the original Task.java file uses title as the String name. A little confusing!

Reply:
Have you been able to test that this works? It seems to me like if you're setting the attribute "comment", you need to have an attribute in this java object called "comment".

Even though it was originally titled "Title" in the original file, I think it would be less confusing to name the variables the same (either comment or title). @zhanche-cp do you have thoughts?

Reply:
Good catch. I think Silas just wants "long - String - long" constructor. So the names are a little inconsistent. Maybe Silas can update the name to make them a little more consistant in next CL?

Reply:
Ah okay, that makes sense. As far as I know it works, but I agree now that the name does seem confusing being named "title" instead of "comment". 
Done!